requirements-test.in

```diff
@@ -7,4 +7,4 @@ flake8-docstrings==1.6.0
 jsonschema
 pytest
 pytest-cov
-pyyaml
+pyyaml
```

Comment:
Please, add a new line in the end of file.

Reply:
Done.